doc/sphinx/src/outputs.rst

```diff
@@ -158,6 +161,134 @@ This will produce a text file (``.hst``) output file every 1 units of
+   num_histograms = 2    # required, specifies how many histograms are defined in this block
+
+   # 1D histogram ("standard", i.e., counting occurance in bin)
+   hist0_ndim = 1
+   hist0_x_variable = advected
+   hist0_x_variable_component = 0
+   hist0_x_edges_type = log
+   hist0_x_edges_num_bins = 10
+   hist0_x_edges_min = 1e-9
+   hist0_x_edges_max = 1e0
+   hist0_binned_variable = HIST_ONES
+
+   # 2D histogram of volume weighted variable according to two coordinates
+   hist1_ndim = 2
+   hist1_x_variable = HIST_COORD_X1
+   hist1_x_edges_type = list
+   hist1_x_edges_list = -0.5, -0.25, 0.0, 0.25, 0.5
+   hist1_y_variable = HIST_COORD_X2
+   hist1_y_edges_type = list
+   hist1_y_edges_list = -0.5, -0.1, 0.0, 0.1, 0.5
+   hist1_binned_variable = advected
+   hist1_binned_variable_component = 0
+   hist1_weight_by_volume = true
+   hist1_weight_variable = one_minus_advected_sq
+   hist1_weight_variable_component = 0
+
+with the following parameters
+
+- ``num_histograms=INT``
+   The number of histograms defined in this block.
+   All histogram definitions need to be prefix with ``hist#_`` where ``#`` is the
+   histogram number starting to count from ``0``.
+   All histograms will be written to the same output file with the "group" in the
+   output corresponding to the histogram number.
+- ``hist#_ndim=INT`` (either ``1`` or ``2``)
+   Dimensionality of the histogram.
+- ``hist#_x_variable=STRING`` (variable name or special coordinate string ``HIST_COORD_X1``, ``HIST_COORD_X2``, ``HIST_COORD_X3`` or ``HIST_COORD_R``)
+   Variable to be used as bin. If a variable name is given a component has to be specified, too,
+   see next parameter.
+   For a scalar variable, the component needs to be specified as ``0`` anyway.
+   If binning should be done by coordinate the special strings allow to bin by either one
+   of the three dimensions or by radial distance from the origin.
+- ``hist#_x_variable_component=INT``
+   Component index of the binning variable.
+   Used/required only if a non-coordinate variable is used for binning.
+- ``hist#_x_edges_type=STRING`` (``lin``, ``log``, or ``list``)
+   How the bin edges are defined in the first dimension.
+   For ``lin`` and ``log`` direct indexing is used to determine the bin, which is significantly
+   faster than specifying the edges via a ``list`` as the latter requires a binary search.
+- ``hist#_x_edges_min=FLOAT``
+   Minimum value (inclusive) of the bins in the first dim.
+   Used/required only for ``lin`` and ``log`` edge type.
+- ``hist#_x_edges_max=FLOAT``
+   Maximum value (inclusive) of the bins in the first dim.
+   Used/required only for ``lin`` and ``log`` edge type.
+- ``hist#_x_edges_num_bins=INT`` (must be ``>=1``)
+   Number of equally spaced bins between min and max value in the first dim.
+   Used/required only for ``lin`` and ``log`` edge type.
+- ``hist#_x_edges_list=FLOAT,FLOAT,FLOAT,...`` (comma separated list of increasing values)
```

Comment:
Does this list include the innermost and outmost edges?

Reply:
Yes, I'll update the `doc` to make this clear.